transport/http/inbound_example_test.go

```diff
@@ -88,28 +87,27 @@ func ExampleMux() {
 func ExampleInterceptor() {
 	// import nethttp "net/http"
 
-	// We create an interceptor which executes an override http.Handler if
-	// the HTTP request is not a well-formed YARPC request.
-	override := func(overrideHandler nethttp.Handler) http.InboundOption {
-		return http.Interceptor(func(transportHandler nethttp.Handler) nethttp.Handler {
-			return nethttp.HandlerFunc(func(w nethttp.ResponseWriter, r *nethttp.Request) {
-				if r.Header.Get("RPC-Encoding") == "" {
-					overrideHandler.ServeHTTP(w, r)
-				} else {
-					transportHandler.ServeHTTP(w, r)
-				}
-			})
-		})
-	}
-
-	// This handler will be executed for non-YARPC HTTP requests.
-	handler := nethttp.HandlerFunc(func(w nethttp.ResponseWriter, req *nethttp.Request) {
+	// Given a fallback http.Handler
+	fallback := nethttp.HandlerFunc(func(w nethttp.ResponseWriter, r *nethttp.Request) {
 		io.WriteString(w, "hello, world!")
 	})
 
-	// We create the inbound, attaching the override interceptor.
+	// Create an interceptor that falls back to a handler when the HTTP request is
+	// missin the RPC-Encoding header.
```

Comment:
missing

Reply:
You can take the boy out of Louisiana...